src/Sulu/Bundle/ContentBundle/Resources/views/Template/content-types/text_line.html.twig

```diff
@@ -1,7 +1,14 @@
+{% set bold = false %}
```

Comment:
hmm i think you can only do this: 
```
{% if params.bold is defined %}
{% set bold = params.bold %}
{% else %}
{% set bold = false %}
{% endif %}
```

if you also define a defaultParam in the content type with `bold = false` then you can get rid of the `if is defined `